vue.config.js

```diff
@@ -2,7 +2,7 @@ const nodeExternals = require("webpack-node-externals");
 
 module.exports = {
   configureWebpack: config => {
-    if (process.env.STYLEGUIDE) {
+    if (process.env.INTERNAL_DEPS) {
```

Comment:
In VueJS, when building a library we externalize all of the node modules, as they're brought in as dependencies.

When developing locally, we want all `INTERNAL_DEPS` to be bundled, so we skip the nodeExternals() plugin.

This repurposes an old flag of the STYLEGUIDE which is no longer in use, but did the same thing